packages/components/src/ui/flex/flex.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Internal dependencies
+ */
+import { createComponent } from '../utils';
+import { useFlex } from './use-flex';
+
+/**
+ * `Flex` is a primitive layout component that adaptively aligns child content horizontally or vertically. `Flex` powers components like `HStack` and `VStack`.
+ *
+ * `Flex` is used with any of it's two sub-components, `FlexItem` and `FlexBlock`.
+ *
+ *
+ * @example
+ * ```jsx
+ * import { Flex, FlexItem, FlexBlock, Text, View } from `@wordpress/components`
```

Comment:
Minor update change the import path to `components/ui`

Reply:
Alternatively we could just remove the import statements from the inline examples. They're probably the least important part of the examples.